weasyl/templates/help/community.html

```diff
@@ -22,20 +22,29 @@ <h4>A. Right to Submit</h4>
+          <p>A “Character” is a drawn or described depiction of a conscious being (e.g., a furry, an anthropomorphized entity, an animal, a human, a robot, or any combination of these) in the submission. A “Cub” (sometimes called a “Babyfur” or “kidfur”) is any character with recognizably humanoid body proportions of children or infants (see below). We are using the term “cub” here as most characters on Weasyl are anthropomorphic animals. NOTE: Not to be confused with a “minor” (i.e., a user of the site who is under the real-life age of 18 and restricted to viewing General content).</p>
+          <p>A “Rating”: All submissions must, at the time of upload, be assigned one of three ratings: General, Mature, or Explicit; content must be rated appropriately.</p>
+          <p>What “Sexual Content” Is: Submissions may include sexually explicit content, either implied or explicit, if rated appropriately. For the purposes of this document, “sexually explicit content” is anything involving but not necessarily limited to: exposed or aroused genitalia, including genital bulges/”tents” or partial concealment of aroused genitals, frotting, masturbation with one character or any form of sexual intercourse between two or more characters, or the depiction or use of sex toys. Furthermore, characters in provocative “pinup” poses who are presenting their genital or perianal regions in a provocative manner can be sexual content, even if those genital regions are concealed. Hyper-enlarged genitals or breasts in any context are sexual content. Certain fetishes and kinks such as BDSM qualify as sexual or explicit content. <em>The final decision of what constitutes “sexual content” or what is merely suggestive is determined by Weasyl’s moderators.</em></p>
+        </li>
+      </ul>
 
       <ol>
         <li><a name="ratings"></a><strong>Ratings</strong>
-          <p>Content ratings should be made based on the context of your submission and its intended audience, as well as on the content of descriptions, tags, and comments supplied by the uploader. We ask that you please rate your content appropriately. For more information on rating your content, please see our <a href="/help/ratings">Ratings Guidelines</a>.</p>
+          <p>This is a brief summary, however, please see the <a href="/help/ratings">Ratings Guidelines</a> for more detailed information.</p>
           <ul>
             <li><strong>General</strong>
-              <p>Content categorized as General is appropriate for all audiences of all ages. It may contain at most very mild themes.</p>
+              <p>Submissions rated “General” are appropriate for all audiences of all ages and may contain mild suggestive content but no frontal nudity, or sexual acts. Certain non-sexual kinks and fetishes are rated General.</p>
```

Comment:
We might want to be clearer about what we mean by nudity. QED, if someone posts a drawing of Lola Bunny from Spacejam in basketball gear - I'd say even with her bellybutton showing it'd be General. Via a literal interpretation of this, it might end up in Mature on here, but the film was rated PG in the US and U (all ages) in the UK...

Open question also about which kinks we declare as "non-sexual"... the clothes related ones especially can be contentious, and rope bondage can be considered an extension of that. I can see it being a point of endless debate.